Kong/docs.konghq.com
# Archive Gateway documentation before 2.x

### Summary
Archiving unsupported Kong Gateway versions.

### Reason
The versions are no longer supported by Kong. The documentation will still exist, but we aren't actively maintaining it. 

### Testing
Netlify preview TBA

* Check the link to the archive in the version dropdowns, this link should go to the Netlify preview site from https://github.com/Kong/docs.konghq.com/pull/3877.
* Check that existing versions of the docs are not affected.
